lms/static/styles/frontend_apps.scss

```diff
@@ -35,6 +31,12 @@
 
 @use 'variables' as var;
 
+// Base is needed for transform base styles. The reset (preflight) that
+// is enabled with the base layer by default is disabled in the tailwind
+// configuration for this project.
+@tailwind base;
```

Comment:
During this process, this was the only "thing" I ran into that threw me for a bit. I was expecting to only need to use the `utilities` layer of tailwind. But we need `base` to get transforms. Including this directive here does not affect output CSS, but makes transforms available. See the local tailwind configuration for the disabling of the reset styles that would, by default, come with the base layer.